### ISSUE TITLE
Create index.js

### ISSUE THREAD

**Issue by Micah456** [Micah456/MyRoutine2#6]
> Dependent on #3 and #5

- [x] If not already created, create static folder
- [x] Create index.js
- [x] Link to index.html
- [x] Load routine names into the html using fetch calls to get data
- [x] Make each button holding a routine, link to the correct address (opening a routine)
- [x] Make sure create button linked to correct page

**Comment by Micah456** [comment]
Consider if unit tests are needed. Create a new issue if tests needed

**Comment by Micah456** [comment]
Also consider turning the buttons into actual buttons to allow for event listening

**Comment by Micah456** [comment]
> Consider if unit tests are needed. Create a new issue if tests needed

Fairly simple functions, no tests needed

**Comment by Micah456** [comment]
> Also consider turning the buttons into actual buttons to allow for event listening

Done, although only the create routine button has an event listener. The rest make use of onclick attribute.
